////////////////////////////////////////////////////////////////////////////////////////////////////
// @breif C_WaitingRoomOutHandler class
////////////////////////////////////////////////////////////////////////////////////////////////////


#pragma once
#include "Packet/Protocol.pb.h"


////////////////////////////////////////////////////////////////////////////////////////////////////
// @breif C_WaitingRoomOutHandler class
////////////////////////////////////////////////////////////////////////////////////////////////////
class C_WaitingRoomOutHandler
{
public:
	// HandlerRun
	static AtBool Handle( PacketSessionPtr& session, C_WaitingRoomOut& pkt );
};
